### PR TITLE
docs(i18n): bunch of fixes

### PR DESCRIPTION
**Pull request by zewa666**
# Pull Request

### CODE REVIEW

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 1]
```diff
@@ -40,28 +40,28 @@
...
 
 Under the hood it uses [i18next](http://i18next.com/), which is a generalized open source library with an extensive set of features.
-By building on top of it not only can you reuse your work across various other platforms and frameworks but you are able to use an extensive eco-system full of various packages.
+By building on top of it not only can you reuse your work across various other platforms and frameworks but you are able to use an extensive eco-system full of various packages and plugins.
 
-If you already familiar with `i18next` you would know that a key along with an optional options object, to manipulate the translation are used to produce the final translated output.
-`@aurelia/i18n` also uses similar concept of key and options object.
-The evaluated keys and the options object are passed on to `i18next` as-is.
-This means that any resource syntax (interpolation, context, nesting etc.), and options object schema supported by `i18next` are also supported by `@aurelia/i18n` by default.
+If you already familiar with `i18next` you would know that a key along with optional options, to manipulate the translation, are used to produce the final translated output.
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: I missed an 'are' here

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 2]
```diff
@@ -84,13 +84,13 @@ new Aurelia()
 
 The above example shows how to initialize the i18n plugin and thereby i18next with translation resources.
 There are alternatives of doing that, which is discussed [later](#managing-translation-resources).
-Once registered, the plugin can be used in view using the translation attribute (default attribute is `t`, though [alias can be configured](#customize-translation-attribute-alias)), and the translation keys configured.
+Once registered, the plugin can be used in your view using the translation attribute (the default attribute name is `t`, though an [alias can be configured](#customize-translation-attribute-alias)), and the translation keys configured.
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: Can you please update the link as well?

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 3]
```diff
@@ -165,17 +165,19 @@ new Aurelia()
...
 
-The registered aliases can then be used in view in place of `t`, as shown below.
+The registered aliases can then be used in your view in place of `t`, as shown below.
 
 ```html
 <element i18n="key1"></element>
 <element tr="key2"></element>
 ```
 
+> Note that you can mix and match any alias at the same time together
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: That is true :)

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 4]
```diff
@@ -189,11 +191,12 @@ If you are unfamiliar such translation resources, it takes following form.
   }
 }
 ```
-For this example, the key expression `key` will be evaluated to `'value of key'`, whereas `complex.nested.resource.key` will be evaluated to `'value of nested resource key'`.
+For this example, the key expression `key` will be evaluated to `'value of key'`, whereas `complex.nested.resource.key` will be evaluated to `'value of nested resource key'`. Nesting resources in this way can help to group translations for individual sections or components.
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: OCD: Missing a new line for new sentence. Easier to comment out part of the text.

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 5]
```diff
@@ -371,7 +375,7 @@ In this case, the `src` attribute of the `img` is replaced instead.
 ```html view.html
 <img t="key">
 ```
-`i18n` plugin transforms the `img` element to `<img src="/path/to/image.jpg">`.
+the `i18n` plugin transforms the `img` element to `<img src="/path/to/image.jpg">`.
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: 't' needs to be capitalized

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 6]
```diff
@@ -600,8 +604,8 @@ export class MyDemoVm {
 
 #### Handling missing keys
 
-Note that by default the key will be rendered instead of value in case the key is missing in the translated resources.
-This is a sensible default as it immediately shows on view that translation is missing for certain keys.
+If a key is missing in the translation resources, by default the key's name will be rendered instead of the value (undefined -> empty string).
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: > undefined -> empty string

this part is not clear in this context

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 7]
```diff
@@ -721,19 +725,19 @@ export class MyDemoVm {
...
 The formatting options are used to affect how the date is formatted.
 For a full list of options look [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat#Parameters).
 
-The value being formatted does not need to be strictly a date object.
-Apart from `Date` instance, both the ValueConverter and the BindingBehavior supports integer, integer string, and ISO 8601 date string as input.
-In case, the input cannot be converted reliably to an instance of `Date`, the original input is returned as-is from these.
-Note that a integer string or a integer input is considered as the number of milliseconds since the Unix epoch (for more details look [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Syntax)).
+The value being formatted does not strictly need to be a date object.
+Apart from `Date` instance, both the ValueConverter and the BindingBehavior support integer, integer strings, and ISO 8601 date string as input.
+In case, the input can't be converted reliably to an instance of `Date`, the original input is returned as-is.
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: > can't

Maybe avoid short form for non-native readers?

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 8]
```diff
@@ -766,14 +770,14 @@ export class MyDemoVm {
...
-This can be useful if you want to cache the `Intl.DateTimeFormat` instance and reuse that later.
+This can be useful if you want to cache the `Intl.DateTimeFormat` instance and reuse it later.
 
 > Note
 > The `I18N#df` in the previous version of Aurelia matches the `I18nService#createDateTimeFormat`, whereas `I18nService#df` provides the formatted date instead.
 
 ### Relative time formatting
 
-`@aurelia/i18n` plugin provides relative time formatting service using [`Intl` API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat).
+`@aurelia/i18n` plugin provides relative time formatting using [`Intl` API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat).
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: Missing a "The" at the beginning?

File: docs/user-docs/2. app-basics/20. internationalization.md [hunk 9]
```diff
@@ -838,11 +842,11 @@ export class MyDemoVm {
 }
 ```
 
-Both ValueConverter and BindingBehavior reacts to this signal and updates the view with the formatted value.
+Both ValueConverter and BindingBehavior react to this signal and update the view with the formatted value.
 
 #### Relative time format via code
 
-Translating date via code works by using the method `I18nService#rt`.
+Translating relative dates via code works by using the method `I18nService#rt`.
```
Comment (Sayan751) [inline docs/user-docs/2. app-basics/20. internationalization.md]: "Translating" should be replaced with "Formatting"

File: docs/user-docs/2. app-basics/20. internationalization.md (hunk 2 above)
Comment (zewa666) [reply docs/user-docs/2. app-basics/20. internationalization.md]: good catch

File: docs/user-docs/2. app-basics/20. internationalization.md (hunk 6 above)
Comment (zewa666) [reply docs/user-docs/2. app-basics/20. internationalization.md]: yep and its btw also explained properly after the sample

File: docs/user-docs/2. app-basics/20. internationalization.md (hunk 9 above)
Comment (zewa666) [reply docs/user-docs/2. app-basics/20. internationalization.md]: I just thought about translating is the wrong word but for the hell I couldn't think about formatting for whatever reason. Good catch

**Comment by zewa666** [comment]
one ~~can't~~ cannot review docs enough I guess ;)

**Comment by Sayan751** [comment]
@zewa666 Thanks :) Merged

**Pull request merged**